contents/blog/avo-plugin-announcement.md

```diff
@@ -0,0 +1,30 @@
+---
+date: 2022-04-26
+title: Introducing the Avo Inspector plugin
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["Release notes", "Product updates"]
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+---
+
+We're excited to announce that we have launched a new plugin with [Avo](https://www.avo.app/), a leading data governance platform which enables you track data consistently and scalably across multiple platforms. 
+
+Connecting PostHog to Avo is great if, for example, you're worried about data going missing due to naming issues or because event details aren't reliably maintained. 
+
+The new plugin, which is available immediately for users on PostHog Cloud or self-hosted deployments, works by streaming event data to the Avo Inspector as it is ingested. Once the plugin is connected, Avo reads the event schemas we send so that you can monitor the quality of your tracking without any code changes. Simple.
```

Comment:
event data > event schema. Good to be clear here that Avo will not receive any user data.